File: src/main/resources/output/IFC4X3_js/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcHalfSpaceSolid/IfcBoxedHalfSpace/IfcBoxedHalfSpace.js
class IfcBoxedHalfSpace extends IfcHalfSpaceSolid {
    constructor() {
        /** @type {IFCBOUNDINGBOX} */
        this.Enclosure = null;
    }

    // === WHERE CLAUSES ===
    // UnboundedSurface : NOT ('IFC4X3_DEV_73740fe4.IFCCURVEBOUNDEDPLANE' IN TYPEOF(SELF\IfcHalfSpaceSolid.BaseSurface))
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcHalfSpaceSolid/IfcBoxedHalfSpace/IfcBoxedHalfSpace.py
class IfcBoxedHalfSpace(IfcHalfSpaceSolid):
    def __init__(self):
        self.Enclosure: IfcBoundingBox = None

    # === WHERE CLAUSES ===
    # UnboundedSurface : NOT ('IFC4X3_DEV_73740fe4.IFCCURVEBOUNDEDPLANE' IN TYPEOF(SELF\IfcHalfSpaceSolid.BaseSurface))
